redbot/core/bot.py

```diff
@@ -406,14 +406,57 @@ async def allowed_by_whitelist_blacklist(
 
         return True
 
-    async def ignored_channel_or_guild(self, ctx: commands.Context) -> bool:
+    async def message_is_eligible_as_command(self, message: discord.Message) -> bool:
```

Comment:
This method name would be better - it's common convention and existing methods in `Red` use same pattern:
```suggestion
    async def is_message_eligible_as_command(self, message: discord.Message) -> bool:
```

Reply:
Alternatively the name from PR title could be used since there's some precedence of that in `Red` too:
```suggestion
    async def message_eligible_as_command(self, message: discord.Message) -> bool:
```